utils/s2n_random.c

```diff
@@ -128,65 +130,73 @@ S2N_RESULT s2n_get_mix_entropy(struct s2n_blob *blob)
     return S2N_RESULT_OK;
 }
 
-void s2n_on_fork(void)
-{
-    zero_if_forked = 0;
-}
-
-static inline S2N_RESULT s2n_defend_if_forked(void)
+static S2N_RESULT s2n_initialise_drbgs(void)
```

Comment:
Let's just do s2n_drbgs_init, to avoid questions of spelling consistency and match the methods in [s2n_init](https://github.com/aws/s2n-tls/blob/main/utils/s2n_init.c#L52-L60).